content-elements/tce-table/edit/index.vue

```diff
@@ -81,6 +82,9 @@ export default {
     findRow(cellId, rows = this.rows) {
       return find(rows, row => row.cells[cellId]);
     },
+    focusElement(cell) {
```

Comment:
Element are receiving scoped element bus. Global event bus shouldn't be used (since Tailor internals should be fully unaccessible).